.github/workflows/graalvm.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           version: ${{ matrix.graalvm }}
           java-version: ${{ matrix.java }}
           components: 'native-image'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Trying to avoid graalvm build failures that are in increase recently, this should help it